state/src/impls/top_level.rs

```diff
@@ -383,12 +374,9 @@ impl TopLevelState {
     ) -> StateResult<()> {
         match action {
             Action::Pay {
-                receiver,
-                quantity,
-            } => {
-                self.transfer_balance(sender_address, receiver, *quantity)?;
-                Ok(())
-            }
+                receiver: _,
```

Comment:
It is just a comment but we were using the pattern `Action::Pay {..}` when ignoring some fields of structs.